common/init.base.rc

```diff
@@ -48,6 +48,10 @@ on init
     copy /dev/cpuset/cpuset.cpus /dev/cpuset/top-app/cpuset.cpus
     copy /dev/cpuset/cpuset.mems /dev/cpuset/top-app/cpuset.mems
 
+    # Support legacy paths for usb storage
```

Comment:
This is not the right place to put this change. sdcard0 is not USB

I suggest not to merge this patch.